### PR TITLE
change JSON field names to camelCase

### PR DESCRIPTION
**Pull request by ydayagi**
https://issues.redhat.com/browse/ECOPROJECT-253
Signed-off-by: Yaron Dayagi <ydayagi@redhat.com>

### CODE REVIEW

**Review by jakub-dzon — CHANGES_REQUESTED** [review]
Please run `make generate manifests` to update generated files.

Make sure that https://github.com/jakub-dzon/k4e-operator/blob/35f1efee0d5f61e376db6e5d81e7028443a128ce/internal/yggdrasil/yggdrasil.go#L259  
works correctly after the changes; I think it might be broken.

File: swagger.yaml [hunk 1]
```diff
@@ -26,14 +26,14 @@ securityDefinitions:
...
-  /data/{device_id}/in:
+  /data/{deviceId}/in:
     get:
       operationId: GetDataMessageForDevice
       tags:
         - yggdrasil
       parameters:
         - in: path
-          name: device_id
+          name: deviceId
```
Comment (jakub-dzon) [inline swagger.yaml:36]: Changes in swagger break compatibility with device-worker, which sends payload in that model.

Comment (ydayagi) [reply swagger.yaml:36]: Are u saying I should not change swagger?

**Comment by ydayagi** [comment]
> Please run `make generate manifests` to update generated files.
> 
> Make sure that
> 
> https://github.com/jakub-dzon/k4e-operator/blob/35f1efee0d5f61e376db6e5d81e7028443a128ce/internal/yggdrasil/yggdrasil.go#L259
> 
> 
> works correctly after the changes; I think it might be broken.

u mean device-worker and not Yggdrasil, correct?

File: swagger.yaml (hunk 1 above)
Comment (jakub-dzon) [reply swagger.yaml:36]: Yes. Unless you can provide compatibility with the older model.

**Comment by jakub-dzon** [comment]
> > Please run `make generate manifests` to update generated files.
> > Make sure that
> > https://github.com/jakub-dzon/k4e-operator/blob/35f1efee0d5f61e376db6e5d81e7028443a128ce/internal/yggdrasil/yggdrasil.go#L259
> > 
> > works correctly after the changes; I think it might be broken.
> 
> u mean device-worker and not Yggdrasil, correct?

Operator. 
device-worker sends hardware information to operator in model defined by swagger.yaml and https://github.com/jakub-dzon/k4e-operator/blob/35f1efee0d5f61e376db6e5d81e7028443a128ce/internal/yggdrasil/yggdrasil.go#L259 function maps swagger model to CR model. All ` utils.Copy` calls might be problematic, because they marshal one data type to JSON and unmarshall it to another.

**Pull request closed without merging**